tools/node_maintain.py

```diff
@@ -0,0 +1,220 @@
+import requests
```

Comment:
maybe /src/utilities is a better place to keep it?

Reply:
Should be in the path ```src/```

Reply:
`src/tools` seems OK. Then move it to `/deployment/paiLibrary` as integrated into paictl.

Reply:
moved to `src/tools`